src/server/upload-and-scan/helpers/handle-file.js

```diff
@@ -59,15 +60,40 @@ async function handleFile(
     ...contentType,
     ...filename
   }
+
+  // Reject zero length files
+  if (uploadResult.fileLength === 0) {
+    files.fileStatus = fileStatus.rejected
+    files.hasError = true
+    files.errorMessage = fileErrorMessages.empty
+  }
+
+  // Reject file if its too big
+  if (uploadResult.fileLength > uploadDetails.maxFileSize) {
+    files.fileStatus = fileStatus.rejected
+    files.hasError = true
+    // TODO: is there a lib to round to the nearest unit
```

Comment:
See the git history. Ill find it

Reply:
Was previously doint something messy with Math.floor and multiplying by 1024 or dividing. But it was messy.

Reply:
Ok. git history has been remarkebly unhelpful.

Reply:
https://github.com/DEFRA/cdp-uploader/blob/0.48.0/src/server/upload/controller.js#L166

Reply:
Had to keep your tongue straight on what was bytes, Kb and Mb with the correct multiplication or division.

Reply:
ended up using the filesize lib to generate this